org.bzdev.CVRDecode.yml

```diff
@@ -0,0 +1,69 @@
+app-id: org.bzdev.CVRDecode
+runtime: org.freedesktop.Platform
+runtime-version: "20.08"
+command: cvrdecode.sh
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+   - org.freedesktop.Sdk.Extension.openjdk11
+modules:
+   - name: openjdk
+     buildsystem: simple
+     build-commands:
+        - /usr/lib/sdk/openjdk11/install.sh
+   - name: cvrdecode
+     buildsystem: simple
+     build-commands:
+       - install -D cvrdecode.sh /app/bin/cvrdecode.sh
+       - install -D cvrdecode.jar /app/cvrdecode.jar
+       - install -D libbzdev-base.jar /app/libbzdev-base.jar
+       - install -D libbzdev-desktop.jar /app/libbzdev-desktop.jar
+       - install -D core.jar /app/core.jar
+       - install -D javase.jar /app/javase.jar
+       - install -D org.bzdev.CVRDecode.desktop /app/share/applications/org.bzdev.CVRDecode.desktop
+       - install -D appicon256.png /app/share/icons/hicolor/256x256/apps/org.bzdev.CVRDecode.png
+       - install -D appicon128.png /app/share/icons/hicolor/128x128/apps/org.bzdev.CVRDecode.png
+       - install -D appicon64.png /app/share/icons/hicolor/64x64/apps/org.bzdev.CVRDecode.png
+       - install -D appicon48.png /app/share/icons/hicolor/48x48/apps/org.bzdev.CVRDecode.png
+       - install -D appicon32.png /app/share/icons/hicolor/32x32/apps/org.bzdev.CVRDecode.png
+       - install -D appicon16.png /app/share/icons/hicolor/16x16/apps/org.bzdev.CVRDecode.png
+       - install -D appdata.xml /app/share/metainfo/org.bzdev.CVRDecode.metainfo.xml
+       - install -D manpage /app/manpage
+     sources:
+        - type: file
+          path: cvrdecode.sh
+        - type: file
+          path: cvrdecode.jar
```

Comment:
where does it download this from? "file" without a URL mean it's in the repository.

Reply:
Sorry about not including the files - I misinterpreted the instructions as requesting only the manifest.  I just pushed all the files it needs on the branch org.bzdev.CVRDecode

Reply:
No, you were absolutely correct. Flathub just hosts and builds the application manifests and brings the source code or binaries in from hosting elsewhere (which could be a github repo in another account, or anything available on the internet).